crates/biome_js_analyze/src/lint/nursery/no_static_element_interactions.rs

```diff
@@ -0,0 +1,353 @@
+                        .collect::<Vec<String>>()
+                } else {
+                    vec![initializer.syntax().text().to_string()]
+                }
+            } else {
+                vec!["true".to_string()]
+            };
+
+            defined_attributes.entry(name).or_insert(values);
+        }
+    }
+    Some(defined_attributes)
+}
+
+fn has_truthy_ariahidden_attr(node: &AnyJsxElement) -> bool {
+    return node
+        .find_attribute_by_name("aria-hidden")
+        .map_or(false, |attr| {
+            attr.as_static_value()
+                .map_or(true, |val| val.text() == "true")
+        });
+}
+
+/// This function allows non-interactive Roles.
+/// That is because these are Roles enabled in eslint-plugin-jsx-a11y.
+fn has_valid_role(role: &str) -> bool {
+    matches!(
+        role,
+        "columnheader"
+            | "form"
+            | "rowheader"
+            | "treeitem"
+            | "alert"
+            | "alertdialog"
+            | "application"
+            | "article"
+            | "banner"
+            | "cell"
+            | "complementary"
+            | "contentinfo"
+            | "definition"
+            | "dialog"
+            | "directory"
+            | "document"
+            | "feed"
+            | "figure"
+            | "grid"
+            | "group"
+            | "heading"
+            | "img"
+            | "list"
+            | "listitem"
+            | "log"
+            | "main"
+            | "marquee"
+            | "math"
+            | "menu"
+            | "menubar"
+            | "navigation"
+            | "note"
+            | "radiogroup"
+            | "region"
+            | "rowgroup"
+            | "section"
+            | "search"
+            | "status"
+            | "table"
+            | "tablist"
+            | "tabpanel"
+            | "term"
+            | "timer"
+            | "toolbar"
+            | "tooltip"
+            | "tree"
+            | "treegrid"
+    )
+}
+
+/// This function disables interactive elements.
+/// This is because this is an element that is disabled by eslint-plugin-jsx-a11y.
+fn is_invalid_element(element_name: &str) -> bool {
+    match element_name {
+        // These cases are interactive with the is_not_interactive_element method,
+        // but is an invalid test case element for eslint-plugin-jsx-a11y.
+        "link" | "header" => true,
+        "area" | "b" | "bdi" | "bdo" | "hgroup" | "i" | "u" | "q" | "small" | "data" | "samp"
+        | "acronym" | "applet" | "base" | "big" | "blink" | "center" | "cite" | "col"
+        | "colgroup" | "content" | "font" | "frameset" | "head" | "kbd" | "keygen" | "map"
+        | "meta" | "noembed" | "noscript" | "object" | "param" | "picture" | "rp" | "rt"
+        | "rtc" | "s" | "script" | "source" | "spacer" | "strike" | "style" | "summary"
+        | "title" | "track" | "tt" | "var" | "wbr" | "xmp" => true,
+        _ => false,
+    }
+}
```

Comment:
I think these aria related methods can be located in `biome_aria` crate with specific context (function naming), because of reusability.

- is_invalid_element
- is_valid_element

cc: @nissy-dev

Reply:
@unvalley 
These methods are:

1. Elements not managed by the aria_roles of Biome
2. Elements that differ between the definition of interactive in Biome and the test cases of eslint-plugin-jsx-a11y
3. Deprecated and obsolete elements

It might be necessary to discuss separately whether they need to be reused across the entire Biome.

Reply:
I see. If this method is for this rule and is unlikely to be used from another rule, then we should leave it here.